tasks/preload.js

```diff
@@ -22,6 +22,7 @@ var maxmin = require('maxmin');
 
 var defaultResourcePatterns = [
 	'**/*.js',
+	'!**/Component-preload.js',
```

Comment:
I'm not sure if it makes sense to add this exclude to the default patterns as they also apply for `libraries`. In this case you might even include a `Component-preload.js` into your library.